apiserver/migrationmaster/facade.go

```diff
@@ -185,3 +257,36 @@ func addToolsVersionForMachine(machine description.Machine, usedVersions map[ver
 		addToolsVersionForMachine(container, usedVersions)
 	}
 }
+
+// tagSlice implements a sensible sort.Interface for []names.Tag
```

Comment:
I've realised this isn't correct anyway because it doesn't take into account container ids nor does it sort unit names correctly. I'm going to tackle it in a different way.